tools/opt/opt.cpp

```diff
@@ -208,6 +206,11 @@ Options (in lexicographical order):
                blank spaces, and in each pair, spec id and default value must
                be separated with colon ':' without any blank spaces in between.
                e.g.: --set-spec-const-default-value "1:100 2:400"
+  --skip-validation
+               Will not validate the SPIR-V before optimizing.  If the SPIR-V
+               is invalid, the optimizer may fail or generate incorrect output.
```

Comment:
Add a warning that states that this should be used with care and only in special circumstances.